src/main/java/org/terasology/gooeyDefence/PathfindingSystem.java

```diff
@@ -75,20 +81,48 @@ public void onChangedBlock(OnChangedBlock event, EntityRef entity) {
         }
     }
 
+    /**
+     * Called to request an enemy be re-pathed.
+     *
+     * @see RepathEnemyRequest
```

Comment:
There can be some comments in RepathEnemyRequest that clarify that it is fired when an entity has strayed off the entrance paths

Reply:
This is now fixed